Content.Server/Speech/Components/BipolarDisorderAccentComponent.cs

```diff
@@ -0,0 +1,31 @@
+namespace Content.Server.Speech.Components;
+
+/// <summary>
+/// Bipolar disorder accent adds random punctuation ("!", "...", "?!") to the end of phrases, and randomly triggers the emotions selected in the component.
+/// </summary>
+[RegisterComponent]
+public sealed partial class BipolarDisorderAccentComponent : Component
```

Comment:
The name doesn't line up with the effects; this reads more like an "unstable emotions" trait. Requires a rename as this risks causing offence.

`EmotionallyUnstableAccentComponent`, `RandomEmotionAccentComponent`, etc.